doc/release/release_0.14.rst

```diff
@@ -0,0 +1,237 @@
+Announcement: scikit-image 0.14.0
+=================================
+
+We're happy to announce the release of scikit-image v0.14.0!
+
+This is the last major release with an official support of Python 2, future
+releases will be developed broadly using Python 3 capabilities, and will be
```

Comment:
will be developed using Python 3 only-syntax. However, 0.14 is a long-term support (LTS) release and will receive bug fixes and backported features deemed important (by community demand) for several years.